### ISSUE TITLE
Multiple partitions show up with no images.

### ISSUE THREAD

**Issue by justin-a30** [files-community/Files#15617]
### Description

Well, it looks like this
![image](https://github.com/files-community/Files/assets/51263008/df4a79a6-849f-46e9-ae5d-dbfcbacbcb35)
This thing seems to be happened randomly while using PC. After I quit completely from tray icon and start again, it seems to be gone. Still, I make this bug report in case.

### Steps To Reproduce

1. Start Files
2. Let it run in background
3. Open it again
4. Multiple partitions show

### Requirements

I expected that even after a long time in background, or using another task, multiple partitions shouldn't be show up.

### Files Version

3.4.1.0

### Windows Version

10.0.22631.3737

### Log File

[debug.log](https://github.com/user-attachments/files/15882999/debug.log)

**Comment by 0x5bfa** [comment]
Thank you for the feedback.

Could you let us know:
- If those empty images are your own images
- If File Explorer shows normally
- Your screen resolution and scale.

**Comment by justin-a30** [comment]
> If those empty images are your own images

No. I didn't do any modifications. I reinstalled Windows recently

> If File Explorer shows normally

Yes, a drive, a CD.

> Your screen resolution and scale.

1366 x 768 - Scale: 100% (default)

**Comment by Josh65-2201** [comment]
Thanks for the feedback, We're tracking this in #14555. You can check there or click subscribe in the right sidebar for updates.

Could you also try turning off Files settings > Advanced > Leave app running in the background when the window is closed > Close and reopen Files.
